arch/install_efistub.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+# Use this as a reference, update loader/initrd according to your kernel/microcode setup
+
+label='rendumOS (efistub)'
```

Comment:
Read label from user? Arch users can just directly run it then, without editing the script

Reply:
editing would be needed for kernel change or microcode additions if any
like if I am using linux-zen with amd-ucode I'd append -zen to loader and initrd and add another initrd=\amd-ucode.img before initramfs
I am unsure how do that on-the-fly (microcode part)

Reply:
Yeah, I just meant for the label
(I'm assuming if random people use the script they'll be on stock kernel, if they're installing custom kernels they'll likely install efistub by themselves, I think)

Reply:
https://github.com/gotenksIN/scripts/pull/5/commits/238c115d87aa4ded50f3d38a91cd311cb6b010c1 this should be enough for that
now microcode handling seems like a bitch, I almost want to ifelse it out but that seems a bit too shitty